### DISCUSSION TITLE
When I press it, the screen is white and I can't see f2 where is my code, what should I do?

### DISCUSSION THREAD

**Discussion by SmailiKsss** [Authenticator-Extension/Authenticator#1250] [Q&A]
When I press it, the screen is white and I can't see f2 where is my code, what should I do?

![2024-08-14_020954](https://github.com/user-attachments/assets/366ff306-8aaa-4452-8d53-8a10e069c71a)

При нажатии белый экран и не видно ф2 где есть мой код как быть

**Comment by Sneezry** [comment]
Are you using Edge? There is a known issue on Edge side. And Microsoft engineers are working on it.

**Reply by SmailiKsss** [reply]
Well, now everything is clear, it turns out we just have to wait until they fix it!
